test/functional/api/window_spec.lua

```diff
@@ -312,6 +312,7 @@ describe('API/win', function()
     end)
 
     it('handles changed buffer', function()
+      command('set nohidden')
```

Comment:
let's also update the test name:

    it("handles changed buffer with 'nohidden'", function()